src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs

```diff
@@ -1,10 +1,11 @@
 ﻿using System.Collections.Generic;
 using Raven.Client.Documents.Conventions;
 using Sparrow.Json;
+using Sparrow.Json.Parsing;
 
 namespace Raven.Client.Documents.Queries.Timings
 {
-    public class QueryTimings : IFillFromBlittableJson
+    public class QueryTimings : IFillFromBlittableJson, IDynamicJsonValueConvertible
```

Comment:
use IDynamicJson and in TrafficWatchHttpChange just do `Timings?.ToJson()`

Reply:
Done.